torch/_inductor/codegen/common.py

```diff
@@ -123,6 +123,14 @@ def bitwise_or(x, y):
     def bitwise_xor(x, y):
         return f"{ExprPrinter.paren(x)} ^ {ExprPrinter.paren(y)}"
 
+    @staticmethod
+    def bitwise_left_shift(x, y):
+        return f"{ExprPrinter.paren(x)} << {ExprPrinter.paren(y)}"
+
+    #    @staticmethod
```

Comment:
Why is the commented out?

Reply:
There is a bug with right shifts in Triton that leads to a test failure.
I just submitted a PR fixing it (https://github.com/openai/triton/pull/1153), which will probably not take long to land, but since it is non-trivial to advance our triton pin, I put some comments in the code and registered a fallback to the aten right shift for now.

Reply:
Mention this in a comment